### PR TITLE
feat: LT-2149 - Add PV Grading and CLC to list of available goods query states

### PR DESCRIPTION
**Pull request by iichr**
- added two additional states for goods queries - PV review and CLC review

https://uktrade.atlassian.net/browse/LT-2149

### CODE REVIEW

File: cases/tests/tests_search_cases.py [hunk 1]
```diff
@@ -22,10 +22,15 @@ def setUp(self):
         super().setUp()
         self.url = reverse("cases:search")
 
+        app_statuses = [
+            app_status[0]
+            for app_status in CaseStatusEnum.choices
+            if app_status[0] not in [CaseStatusEnum.CLC, CaseStatusEnum.PV]
```
Comment (iichr) [inline cases/tests/tests_search_cases.py]: Excluding the two new states which are only applicable to goods queries

File: static/statuses/enums.py [hunk 2]
```diff
@@ -22,6 +22,8 @@ class CaseStatusEnum:
     UNDER_FINAL_REVIEW = "under_final_review"
     UNDER_REVIEW = "under_review"
     WITHDRAWN = "withdrawn"
+    CLC = "clc_review"
+    PV = "pv_review"
```
Comment (iichr) [inline static/statuses/enums.py]: Not really case statuses but neither are closed/submitted/withdrawn, perhaps they don't warrant a good specific status enum

File: cases/tests/tests_search_cases.py (hunk 1 above)
Comment (Mark-Spiers-kainos) [reply cases/tests/tests_search_cases.py]: could you add this as a comment in the code?

Comment (iichr) [reply cases/tests/tests_search_cases.py]: Yeah sure

File: cases/tests/tests_search_cases.py [hunk 3]
```diff
@@ -22,10 +22,15 @@ def setUp(self):
         super().setUp()
         self.url = reverse("cases:search")
 
+        app_statuses = [
+            app_status[0]
+            for app_status in CaseStatusEnum.choices
```
Comment (BenAndrew-Kainos) [inline cases/tests/tests_search_cases.py]: Could we instead make a new list in CaseStatusEnum called something like ApplicationStatuses which has the correct list with PV & CLC removed?

Comment (BenAndrew-Kainos) [reply cases/tests/tests_search_cases.py]: This would be more typical in the system at the moment

Comment (iichr) [reply cases/tests/tests_search_cases.py]: @BenAndrew-Kainos Would it be worth duplicating the enum - 23 status for applications only, just to exclude the 2?

Comment (BenAndrew-Kainos) [reply cases/tests/tests_search_cases.py]: No I mean just define this in the enum instead

Comment (BenAndrew-Kainos) [reply cases/tests/tests_search_cases.py]: It'll be easier to maintain & reuse that way

**Pull request merged**